_posts/2016-10-10-Help!-I'm-becoming-Post-Junior.md

```diff
@@ -0,0 +1,166 @@
+
+Other options are to give [technical talks](http://artsy.github.io/blog/2016/03/09/public-speaking-part1-is-it-for-me/) within your company on specific topics, for example [Licensing for OSS code][11] is a talk I gave internally to Artsy. We now have a weekly team “Lunch & Learn” where we give anyone the chance to talk, or request a talk on a topic. 
+
+This gives a lot of space for personal growth too, as these are really easy to transition into blog posts and meetup talks.
+
+Finally, try to pair with programmers outside of your direct team, it could be on whatever they’re working on - you’ll learn a bit more about other systems and they’ll get to [rubber duck][12] their problems.
+
+## Side Projects
+
+I’ve tried to focus this post specifically on things you can do on work time. Not everyone has the ability to go home and spend a few hours on this and that. However, programming at the early stage can generally be a “you get out what you put in” kind of deal with time.  
+
+Side projects give you the chance to test out new ideas in an isolated environment that is totally under your control. They are great places for exploring what makes a system tick. Some side-projects are built to be [thrown away][13] others can [to last][14] [for years][15]. Both are valuable spaces for experimentation.
+
+## Contributing to Open Source
+
+A lot of people’s work relies on Open Source code and as you are starting to branch out into having a larger impact - perhaps making smaller improvements to the projects you rely on every day could help. This is [how I got started][16] working with larger projects in the Open Source world.
+
+Interacting in these projects exposes you to whole new teams of people with, hopefully, very different perspectives. It will change you, you will change them.
+
+## Wrap-up
+
+This is no simple “one-step to consider yourself post-Junior" article. Only a collection of ideas that you can apply until you feel confident as you find you own ways to help out. As you grow, you grow in many different directions at once - and all of them are valid. 
+
+You can grow by research, practice, doing things outside of work, doing small projects with others in work, experimentation with technology, interacting with more and more people and re-thinking existing approaches. Once you're outside of small feature work - there are so many ways you can contribute.   
+
+We use the idea of an engineering ladder as a yardstick to ensure we treat developers fairly at Artsy. Different companies will have different ways of scoping how you measure up as a programmer. Your company's ladder can help offer direction for what they would love to see.
+
+At the end of the day you’ve got your foundations now, and its time to start thinking about building yourself into a unique programmer and creating your own opinions. There’s never been a better time to start.
+
```

Comment:
^ @mennenia gave this a re-write, do you feel it fits better with the "no one true way" concept I've been trying to push? 

Infact I didn't really find a nice place to put impact in there at all. 

Perhaps realistically, the larger discussion around impact is in a "I've been programming for 4-5 years - where can I go now?" - as that is really where I started to focus on that as a useful metric